doc/_templates/autoapi/python/data.rst

```diff
@@ -6,6 +6,9 @@
 
 {% if module is none %}
 .. py:{{ obj.type }}:: {{ obj.name }}
+   {%+ if obj.docstring == "autoapi_noindex" -%}
```

Comment:
Can we use a contains check instead? I'd like to add documentation to constants eventually.

Reply:
Updated